.github/workflows/update_website.yml

```diff
@@ -33,57 +29,26 @@ jobs:
         run: |
           ls -a
 
-      - name: Clear cloned website repo
+      - name: Clear and setup directories inside kerzendorf-group.github.io
         run: |
-          cd kerzendorf-group.github.io
-          rm -rf index.html Research.html Contact.html People.html Support.html News.html
-          rm -rf assets website_files members sub_research support_images news
-
-      - name: Setup directories inside kerzendorf-group.github.io
-        run: |
-          cd kerzendorf-group.github.io
-          mkdir members sub_research news
-          cp -r ../group-data/website_data/website_files ./
-          cp -r ../groupwebsite_generator/assets ./
-          # rsync -av  --exclude 'jsons' ../group-data/members/ ./members
-
-
-          source_directory="../group-data/members/"
-          destination_directory="members"
-          
-          
-          for dir in "$source_directory"/*; do
-              if [ -d "$dir" ]; then
-                  directory_name=$(basename "$dir")
-                  
-                  if [ -f "$dir/jsons/basic_info.json" ]; then
-                      kg_value=$(jq -r '.display.kg' "$dir/jsons/basic_info.json")
-                      
-                      if [ "$kg_value" == "true" ]; then
-                          rsync -av --exclude 'jsons' "$dir/" "$destination_directory/$directory_name"
-                      fi
-                  fi
-              fi
-          done
+          cd groupwebsite_generator
+          ./notebooks/setup.sh
+      
+      - name: Set up Python3
```

Comment:
I think this should do somewhere earlier in the list of steps, so you do install, and then you run all the scripts. You can also use the woorking directory option to combine a few steps. `cd` makes it seem like you would move into that directory and it would persist but that is not the case.

Reply:
Updated all steps to use working-directory.